docs/guide/setup/defining_protocols.rst

```diff
@@ -3,46 +3,41 @@
 Creating and using Protocols
 ============================
 
-With a thermodynamic cycle of interest identified and represented using `ChemicalSystem` objects,
-the next step is to choose a method for estimating the free energy differences.
-This is done using a :class:`.Protocol`,
-a computational method for estimating the free energy difference between two chemical systems.
-
-Just as there are multiple possible methods for estimating free energy differences,
-there are multiple available Protocols to choose from.
-For example, included in the ``openfe`` package are the
-:class:`.RelativeHybridTopologyProtocol`,
-:class:`.AbsoluteSolvationProtocol`,
-and :class:`.PlainMDProtocol`;
-for a full list see :ref:`userguide_protocols`.
-This selection is being built upon,
-both by the openfe development team as well as with external academic groups.
-
-What all these Protocols share is a common interface for how they are created and executed,
-therefore it is relatively simple to
-try out a new method,
+A :class:`.Protocol` is a computational method for estimating the free energy difference between two chemical systems.
+
+Included in the ``openfe`` are the following common protocols:
+   * :class:`.RelativeHybridTopologyProtocol`
+   * :class:`.AbsoluteSolvationProtocol`
+   * :class:`.PlainMDProtocol`
+
+For more information, see  :ref:`userguide_protocols`.
```

Comment:
The intention was that userguide_protocols was where we would keep an up to date list of Protocols, hence why we were intentionally refering back to it as "look here for a full up-to-date list".